src/Nox.Types.EntityFramework/Abstractions/IEntityDtoSqlQueryBuilder.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Nox.Types.EntityFramework.Abstractions;
+
+public interface IEntityDtoSqlQueryBuilder
+{
+    string EntityName { get; }
+    string Build();
```

Comment:
Add a summary explaining that we are building a custom query to return localized fields from a LocalizedTable and setting up the query in the DbContext for that entity, making this transparent for anyone that queries some entity, having this functionality OOB